generators/typescript/sdk/generator/src/contexts/base-client/BaseClientContextImpl.ts

```diff
@@ -290,7 +290,10 @@ export class BaseClientContextImpl implements BaseClientContext {
             kind: StructureKind.PropertySignature,
             name: "fetch",
             type: "typeof fetch",
-            hasQuestionToken: true
+            hasQuestionToken: true,
+            docs: [
+                "Provide a custom fetch implementation. Useful for environments like React Native that require a custom fetch function."
```

Comment:
Let's not mention specific runtimes or environments, but for any platform that doesn't have fetch, or needs a custom fetch for whatever reason. Make the jsdoc comment generic, more widely applicable.